posthog/tasks/periodic_digest/periodic_digest.py

```diff
@@ -38,10 +38,12 @@
 
 
 @dataclasses.dataclass
-class periodicDigestReport:
+class PeriodicDigestReport:
     new_dashboards: list[dict[str, str]]
     new_event_definitions: list[dict[str, str]]
     new_playlists: list[dict[str, str]]
+    interesting_collections: list[dict[str, str]]
+    interesting_saved_filters: list[dict[str, str]]
```

Comment:
adding the interesting things as separate lists so we can treat them differently if need be